### DISCUSSION TITLE
Demostración Ejercicio 8 TP Árboles AVL

### DISCUSSION THREAD

**Discussion by franualba** [harpomaxx/algoritmos2#6] [General]
Hola Profe! 

Le escribo para preguntarle si nos puede enviar cuando pueda una posible demostración del ejercicio 8 del tp de árboles AVL.

**Comment by harpomaxx** [comment]
Esta es una posible demostración siguiendo un enfoque inductivo. NO hay necesidad de caer en algo tan formal. Se las dejo como un ejemplo.

![image](https://user-images.githubusercontent.com/12446313/226601658-de57673b-5607-475f-83b6-061c0a603760.png)
![image](https://user-images.githubusercontent.com/12446313/226601822-246d4e22-705f-4154-8146-b60a9365abc6.png)
